src/Codegen/Constraints/Context.php

```diff
@@ -130,4 +130,8 @@ public function acknowledgeRef(string $ref): void {
   public function hasSeenRef(string $ref): bool {
     return C\contains($this->seen_refs, $ref);
   }
+
+  public function shouldIgnoreAdditionalProperties(): bool {
```

Comment:
small nit, but should we update this based on the new naming?

Reply:
Great catch! Fixed